lib/pf/domain/ntlm_cache.pm

```diff
@@ -110,9 +164,8 @@ Fetch the NT hashes of all the valid users of a domain
 sub fetch_all_valid_hashes {
     my ($domain) = @_;
     my $logger = get_logger;
+    
     my $config = $ConfigDomain{$domain};
-    my $source = getAuthenticationSource($config->{ntlm_cache_source});
```

Comment:
I'd have to pass the source object in the other subs and change their signatures.

Nothing against that, but I was trying to keep the subroutines with simple parameters since its easier to call them from the command line when you troubleshoot it